docs/usage/sensitivity.rst

```diff
@@ -0,0 +1,212 @@
+- ``descriptio`` : Description in text or HTML format. *Optional*.
+- ``periode``: Numbers of the months in which the area is occupied, **comma separated** and **without spaces** (e.g. ``6,7,8`` for June, July and August).
+- ``practices``: Names of practices, separated by commas, without spaces (e.g. ``Terrestre,Aerien,Vertical``), see :ref:`Practices`. Otherwise, the line import will fail.
+- ``url`` : Record url. *Optional*.
+
+Import
+======
+
+- Click on the **user link** at top right, then on **Imports**,
+- select the type of data to be imported (**species** or **regulatory area**),
+- select the *.zip* file to be imported,
+- select the correct encoding (``UTF8`` or ``Windows-1252``)
+- click on **Import**,
+- wait a few seconds,
+- the import progress is displayed on the right,
+- click on **Display report** to see any unimported lines.
+
+
+.. warning:: 
+  Relaunching an import **with the same file** will create duplicates.
+
+
+#########
+API usage
+#########
+
+.. note::
+
+  You can play with API using Biodiv'Sports widget tool: https://biodivsports-widget.lpo-aura.org/
+
+
+Requesting URLs
+===============
+
+.. _Common Parameters:
+
+Commons parameters
+------------------
+
+
+If ``language`` parameter is provided, api returns directly translated field, else, a dictionnary of traductions is returned
+    
+e.g. ``/api/v2/sensitivearea_practice/1/?``
+
+
+.. code-block:: JSON
+
+    {
+      "id":1,
+      "name":{
+        "fr":"Terrestre",
+        "en":"Land",
+        "it":null
+      }
+    }
+
+
+e.g. ``/api/v2/sensitivearea_practice/1/?language=en``
+
+
+.. code-block:: JSON
+
+    {
+      "id":1,
+      "name":"Land"
+    }
+
+.. _Practices:
+
+Sport practices
+---------------
+
+List of outdoor practices
+
+``/api/v2/sensitivearea_practice/``
+
+e.g. https://biodiv-sports.fr/api/v2/sensitivearea_practice/
+
+
+Sensitive areas
+---------------
+
+List of sensitive areas
+
+``/api/v2/sensitivearea/``
+
+The default output format is ``json``. To obtain output in ``geojson`` format, simply add the ``format=geojson`` parameter.
+
+``/api/v2/sensitivearea/?format=geojson`` 
+
+e.g. https://biodiv-sports.fr/api/v2/sensitivearea/?format=geojson
+
+**Filtering data**
+
+Data can be filtered through those parameters:
+
+- ``language`` : API language (see :ref:`Common Parameters`)
+
+  - Expected values: ``fr``, ``en``, ``es`` or ``it``
+  - e.g. ``/api/v2/sensitivearea/?language=fr``
+
+- ``period`` : Sensitivy period (monthes list)
+
+  - Expected values: List of month number (from 1 to 12), comma separated
+  - e.g. ``/api/v2/sensitivearea/?period=4,5,6,7``
+
+- ``practices`` : Outdoor sport practices
+
+  - Expected values: List of practices ids (see :ref:`Practices`)
+  - e.g. ``/api/v2/sensitivearea/?practices=1,2``
+
+.. - ``structure`` : Organization that declared the sensitive area. 
+
+..   - Expected values: List of practices ids (see :ref:`Structures`)
+..   - e.g. ``/api/v2/sensitivearea/?structure=1,2``
```

Comment:
Is it normal that this section is commented and therefore will not be visible in the documentation ?

Reply:
nope, fixed